packages/toolpad-app/src/toolpad/Apps/index.tsx

```diff
@@ -738,7 +751,33 @@ export default function Home() {
             Apps
           </Typography>
           <FlexFill />
-          <Button onClick={() => setCreateDialogOpen(true)}>Create New</Button>
+          <TextField
```

Comment:
Is there really no other way to style this text field as we want to without these a bit ugly overrides? Can't we do it with just regular props? But I understand that if the intended design is imposing certain things there might be no other way to do it.

I also noticed that the icon and the text are slightly misaligned vertically, but maybe most people wouldn't even notice:
<img width="227" alt="Screenshot 2023-01-04 at 10 24 54" src="https://user-images.githubusercontent.com/10789765/210534895-5c0a0639-307e-4e9d-854e-a13d28c68fce.png">

Reply:
At least we should extract these styles to a variable above the component, so that we don't mix them with the JSX logic.

Reply:
> Is there really no other way to style this text field as we want to without these a bit ugly overrides? 

In the beginning I added these overrides to the default theme so as to make all text fields look more dense and match the buttons in the theme, but in the discussion [here](https://github.com/mui/mui-toolpad/pull/1402#discussion_r1047139408) we decided to go ahead with styling just this text field for now. 

> At least we should extract these styles to a variable above the component, so that we don't mix them with the JSX logic.

Agreed, I'll do this.

> I also noticed that the icon and the text are slightly misaligned vertically, but maybe most people wouldn't even notice:

Agreed, this was much less noticeable in the dark theme. I'll fix this